Automattic/woocommerce-payments
# feat: add dev mode in settings screen

Fixes https://github.com/Automattic/woocommerce-payments/issues/1835

#### Changes proposed in this Pull Request









Adding dev/test mode settings.

#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Go to the WCPay settings: http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Test mode checkbox should be checked/unchecked/disabled

![Screen Shot 2021-05-24 at 10 01 04 AM](https://user-images.githubusercontent.com/273592/119366833-fb1bd480-bc76-11eb-910e-fb314c6fa53a.png)


-------------------
